kiwi/volume_manager/btrfs.py

```diff
@@ -381,7 +381,7 @@ def _set_snapper_sysconfig_file(cls, root_path):
         sysconf_file = SysConfig(
             os.sep.join([root_path, 'etc/sysconfig/snapper'])
         )
-        if not sysconf_file.get('SNAPPER_CONFIGS') or \
+        if not sysconf_file['SNAPPER_CONFIGS'] or \
```

Comment:
As said above I think this should be coded either using the `get` method (for that needs to be implemented in Sysconfig) or just verify key existance with something like:

```python
if 'SNAPPER_CONFIGS' not in sysconf_fikiwi/volume_manager/btrfs.pyle or ...
```